README.md

```diff
@@ -113,7 +113,30 @@ final ManagedCursorManager cursorManager = new ManagedCursorManager(baseUri, req
 ...
 final Subscription subscription = nakadiClient.subscribe("fahrschein-demo2", eventName, "fahrschein-demo-sales-order-placed");
 nakadiClient.listen(subscription, SalesOrderPlaced.class, listener, streamParameters);
+```
+
+## Using multiple partitions and multiple consumers
+
+With the `PartitionManager` api it is possible to coordinate between multiple nodes of one application, so that only one node is consuming events from a partition at the same time.
 
+Partitions are locked by one node for a certain time. This requires that every node has an unique name or other identifier. The `listen` method will return after the lock timeout and can try to grab the lock again.
+
+```java
+@Scheduled(fixedDelay = 60*1000L)
```

Comment:
> Why do you delay this method?

Good point, there are now two cases where this method can exit:

 - after the specified timeout
 - when an unchecked exception is thrown

For the 2nd case a slight delay might make sense. Another way to implement it would be with the `Async` annotation and having another loop inside the method.